xgi/classes/hypergraphviews.py

```diff
@@ -18,7 +18,7 @@
 __all__ = ["subhypergraph"]
 
 
-def subhypergraph(H, nodes=None, edges=None):
+def subhypergraph(H, nodes=None, edges=None, keep_isolates=True):
```

Comment:
The default is True, but the docstring says the default is False!

Reply:
Thanks, just fixed it.